src/views/shared/token-icon/token-icon.view.tsx

```diff
@@ -1,27 +1,17 @@
 import { FC } from "react";
-import tokens from "src/assets/tokens/tokens.json";
+
```

Comment:
We had an issue for this refactor. Maybe we can close the issue now?

I think the only differences are:

* the name of the component was proposed to be `Icon`, being more generic than `TokenIcon` (since this component is no longer related to a Token).
* `logoURI` was proposed as `url`, since `logoURI` is something we get from the `tokens.json`.

Reply:
Right, I've changed the implementation to match the proposal. I'll close the issue when merging this PR.